src/main/java/org/apache/mesos/offer/OfferAccepter.java

```diff
@@ -45,7 +46,7 @@ public OfferAccepter(Set<OperationRecorder> recorders) {
   }
 
   public List<OfferID> accept(SchedulerDriver driver, List<OfferRecommendation> recommendations, Filters filters) {
-    if (recommendations.size() <= 0) {
+    if (CollectionUtils.isEmpty(recommendations)) {
```

Comment:
any reason to not just use recommendations.isEmpty()? (same with operations.isEmpty() below)